taxi/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.views import View
+
+from taxi.models import Driver, Car
+
+
+class LicenseNumberValidatorMixin(forms.ModelForm):
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) == 8:
+            RegexValidator(
+                regex=r"[A-Z]{3}\d{5}",
+                message="Ensure it meets the format of 3 uppercase "
+                        "letters followed by 5 digits. Exemple:'ABC12345'"
+            )(license_number)

```

Comment:
The `RegexValidator` is correctly used to validate the license number format. However, ensure that the regex pattern matches the exact requirements. The pattern `[A-Z]{3}\d{5}` matches three uppercase letters followed by five digits, which seems correct based on the message provided.